repos/system_upgrade/el7toel8/actors/networkmanagerupdateclientid/libraries/networkmanagerupdateclientid.py

```diff
@@ -0,0 +1,66 @@
+import dbus
```

Comment:
Considering that this in the ApplicationsPhase and I can't see a python2-dbus I wonder if will run into some trouble with this.

Reply:
Understood, I couldn't test it due to other issues I had with the upgrade.

So, everything that runs in ApplicationsPhase uses python2, right?

The actor needs to talk with NM to update connection profiles. The option available are:

 1. use D-Bus: we need python2-dbus, which seems not available in RHEL8

 2. use libnm through GObject-introspection: we need python2-gobject not available on RHEL8

 3. use nmcli commands: this is the worst option because it's inefficient and requires additional string	parsing

As I understand it, we're left with 3), right? Or do you think in some ways we could use one of the other two?

Reply:
@bengal We will probably switch into the python3 after the UpgradePhase is done. So you would be able to (ok , you will have to in case we will do that) use Python3 libraries in actor for  `ApplicationsPhase`. But in that case, you need to wait for implementation. Not sure when it would be resolved.

Reply:
Okay, then I'll wait the switch to python 3 for the upgrade phase.